<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>727</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dynamometer by JayTek</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1040</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Configuration_As"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Channels_As"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuConnection">
    <property name="title">
     <string>Connection</string>
    </property>
    <addaction name="actionConnectionSettings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuConnection"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>Tool Bar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionNewChartView"/>
   <addaction name="actionNewDash"/>
   <addaction name="actionChannelEditor"/>
   <addaction name="actionConfiguration"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDockWidget" name="channelDock">
   <property name="windowIcon">
    <iconset resource="dynamometer.qrc">
     <normaloff>:/icons/channels.png</normaloff>:/icons/channels.png</iconset>
   </property>
   <property name="windowTitle">
    <string>Channels</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="channelLineEdit">
           <property name="inputMask">
            <string/>
           </property>
           <property name="maxLength">
            <number>60</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="channelAddPushButton">
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="dynamometer.qrc">
             <normaloff>:/icons/chart_curve_add.png</normaloff>:/icons/chart_curve_add.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="channelDeletePushButton">
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="dynamometer.qrc">
             <normaloff>:/icons/chart_curve_delete.png</normaloff>:/icons/chart_curve_delete.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="channelListWidget">
         <property name="dragEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="setupDock">
   <property name="windowTitle">
    <string>Setup</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>RPM</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Brake</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Auto Run</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="sessionDock">
   <property name="windowTitle">
    <string>Session</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="topMargin">
      <number>9</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>9</number>
     </property>
     <item>
      <widget class="QTableWidget" name="sessionTableWidget">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>24</number>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderMinimumSectionSize">
        <number>24</number>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>24</number>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="fileDock">
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <property name="windowTitle">
    <string>Files</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QTreeView" name="fileTree">
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
       </property>
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::InternalMove</enum>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::MoveAction</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="animated">
        <bool>false</bool>
       </property>
       <property name="expandsOnDoubleClick">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionConnectionSettings">
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Connection Settings</string>
   </property>
   <property name="statusTip">
    <string>connect to dynamometer</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Quit the application</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="dynamometer.qrc">
     <normaloff>:/icons/disconnect.png</normaloff>
     <normalon>:/icons/connect.png</normalon>:/icons/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>Toggle Connection</string>
   </property>
   <property name="toolTip">
    <string>toggle connection to the dynamometer</string>
   </property>
  </action>
  <action name="actionChannelEditor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="dynamometer.qrc">
     <normaloff>:/icons/channels.png</normaloff>:/icons/channels.png</iconset>
   </property>
   <property name="text">
    <string>Edit Channels...</string>
   </property>
   <property name="toolTip">
    <string>open the channel editor</string>
   </property>
  </action>
  <action name="actionNewChartView">
   <property name="icon">
    <iconset resource="dynamometer.qrc">
     <normaloff>:/icons/session.png</normaloff>:/icons/session.png</iconset>
   </property>
   <property name="text">
    <string>New...</string>
   </property>
   <property name="toolTip">
    <string>Create new graph view</string>
   </property>
  </action>
  <action name="actionNewDash">
   <property name="icon">
    <iconset resource="dynamometer.qrc">
     <normaloff>:/icons/dash.png</normaloff>:/icons/dash.png</iconset>
   </property>
   <property name="text">
    <string>New...</string>
   </property>
   <property name="toolTip">
    <string>Create a new dash</string>
   </property>
  </action>
  <action name="actionConfiguration">
   <property name="icon">
    <iconset resource="dynamometer.qrc">
     <normaloff>:/icons/config.png</normaloff>:/icons/config.png</iconset>
   </property>
   <property name="text">
    <string>Configuration</string>
   </property>
   <property name="toolTip">
    <string>Configure Dynamometer</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionSave_Configuration_As">
   <property name="text">
    <string>Save Configuration As...</string>
   </property>
  </action>
  <action name="actionSave_Channels_As">
   <property name="text">
    <string>Save Channels As...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>channelLineEdit</tabstop>
  <tabstop>channelAddPushButton</tabstop>
  <tabstop>channelDeletePushButton</tabstop>
  <tabstop>channelListWidget</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources>
  <include location="dynamometer.qrc"/>
 </resources>
 <connections/>
</ui>
